views/NewChart.react.js

```diff
@@ -1,9 +1,69 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { compose } from 'redux'
+import { withStyles } from '@material-ui/core/styles'
+import { connect } from 'react-redux'
+
+import Header from './components/Header'
+import Sidebar from './components/Sidebar'
+
+import ChartForm from './forms/ChartForm'
+
+import getCounts from './fe-utils/countUtil'
+import getAvatar from './fe-utils/avatarUtil'
+import { fetchSubjects } from './fe-utils/fetchUtil'
+import getDefaultStyles from './fe-utils/styleUtil'
+import { chartStyles } from './styles/chart_styles'
+
+const NewChart = ({ user, classes }) => {
```

Comment:
it doesn't look like `classes` are passed in to this component. Can we remove this prop?